packages/zcli-apps/src/commands/apps/update.ts

```diff
@@ -35,21 +35,30 @@ export default class Update extends Command {
     try {
       const { app_id }: any = await getUploadJobStatus(job_id, appPath)
       cli.action.stop('Deployed')
+      const updatedSummary: string[] = []
 
-      const installations: Installations = await request.requestAPI('/api/v2/apps/installations.json', {}, true)
+      Object.keys(manifest.location).forEach(async product => {
```

Comment:
So we are looping through locations, is this how ZAT does it? My suggestion would be to see if we can do it via single endpoint instead of looping through per product and making multiple calls. But if this is the only way to do it then i guess no choice. 

cc @fbvilela (for ZAM review since they own these endpoints and might have some recommendations)

Reply:
Yes, ZAT loops through locations, see [here](https://github.com/zendesk/zendesk_apps_tools/blob/master/lib/zendesk_apps_tools/command.rb#L201-L204).  It appears that this is the recommended approach by the ZAM group, so closing this thread per said discussion.